app/src/main/java/com/cherrish/android/presentation/challenge/mission/ChallengeMissionViewModel.kt

```
@@ -52,12 +79,25 @@ class ChallengeMissionViewModel @Inject constructor() : ViewModel() {
+                    homecareRoutineId = routineIdArg,
+                    routineNames = currentMissions.map { it.missionContent }
+                )
+                .onSuccess { response ->
+                    _sideEffect.emit(
+                        ChallengeMissionSideEffect.NavigateToChallengeMissionProgress
+                    )
+                }.onLogFailure { }
         }
     }
+
     fun onBackClick() {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)